building `tensorflow-core-ops` broken by some recent change ..

(hey all, sorry i've been ignoring comments on PRs for a few days; i've been moving house!)

when trying to build the master branch from scratch i now get:

```
> stack --docker --docker-image=$IMAGE_NAME build                                                                                                                                         1 ↵
tensorflow-core-ops-0.1.0.0: configure (lib)
Log files have been written to: /home/noon/dev/ext/tensorflow-haskell/.stack-work/logs/
Progress: 1/5
--  While building package tensorflow-core-ops-0.1.0.0 using:
      /home/noon/.stack/programs/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/ghc-7.10.3/bin/ghc --make -odir /home/noon/dev/ext/tensorflow-haskell/tensorflow-core-ops/.stack-work/dist/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/Cabal-1.22.5.0/setup -hidir /home/noon/dev/ext/tensorflow-haskell/tensorflow-core-ops/.stack-work/dist/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/Cabal-1.22.5.0/setup -i -i. -clear-package-db -global-package-db -package-db=/home/noon/.stack/snapshots/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/lts-6.2/7.10.3/pkgdb -package-db=/home/noon/dev/ext/tensorflow-haskell/.stack-work/install/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/lts-6.2/7.10.3/pkgdb -hide-all-packages -package=Cabal-1.22.5.0 -package-id=array-0.5.1.0-960bf9ae8875cc30355e086f8853a049 -package-id=base-4.8.2.0-0d6d1084fbc041e1cded9228e80e264d -package-id=bin-package-db-0.0.0.0-c2998655922e5892be4d23c583233dd1 -package-id=binary-0.7.5.0-bf6a937522bb8cec5d00218470f28789 -package-id=bytestring-0.10.6.0-9a873bcf33d6ce2fd2698ce69e2c1c66 -package-id=containers-0.5.6.2-59326c33e30ec8f6afd574cbac625bbb -package-id=deepseq-1.4.1.1-614b63b36dd6e29d2b35afff57c25311 -package-id=directory-1.2.2.0-660a7a83a753ed85c8a374c15dae2b97 -package-id=filepath-1.4.0.0-f97d1e4aebfd7a03be6980454fe31d6e -package-id=ghc-prim-0.4.0.0-6cdc86811872333585fa98756aa7c51e -package-id=haskeline-0.7.2.1-ee66b80ab55ef13718feae46e4940f5d -package-id=hoopl-3.10.0.2-e1d2f66d9c173061362d378e84471a99 -package-id=hpc-0.6.0.2-a33eebda3a4ae222891fd5d14c4f2b1f -package-id=integer-gmp-1.0.0.0-3c8c40657a9870f5c33be17496806d8d -package-id=lens-family-1.2.0-2ee1455711f137e5934d52960f7c8de9 -package-id=mainland-pretty-0.4.1.4-50285c86e8742407fa4c6ab802a9fa2c -package-id=pretty-1.1.2.0-5cc412214ea63f61ee84c4fbabdbe0ec -package-id=process-1.2.3.0-a22328103774f0499a990653944cbf99 -package-id=proto-lens-0.1.0.4-8f71435f97cecb0d4abb0e4f05221e76 -package-id=template-haskell-2.10.0.0-3c4cb52230f347282af9b2817f013181 -package-id=tensorflow-0.1.0.0-e20ca24aee7f264ccc7941a1a0f7cec4 -package-id=tensorflow-opgen-0.1.0.0-bacacaf1d71fc43e6d64d7edd8d7aed1 -package-id=terminfo-0.4.0.1-ff5b3d5838955b7711e16ccc720f06f4 -package-id=text-1.2.2.1-f5b1651133426aa2c83c19d64d248779 -package-id=time-1.5.0.1-1b9a502bb07a3e6f4d6935fbf9db7181 -package-id=transformers-0.4.2.0-81450cd8f86b36eaa8fa0cbaf6efc3a3 -package-id=unix-2.7.1.0-bb54ee8f9f6b2600aae7a748eb88a610 -package-id=xhtml-3000.2.1-e005917157f780654d68110616d034e8 /home/noon/dev/ext/tensorflow-haskell/tensorflow-core-ops/Setup.hs /home/noon/.stack/setup-exe-src/setup-shim-mPHDZzAJ.hs -main-is StackSetupShim.mainOverride -o /home/noon/dev/ext/tensorflow-haskell/tensorflow-core-ops/.stack-work/dist/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/Cabal-1.22.5.0/setup/setup -threaded
    Process exited with code: ExitFailure 1
    Logs have been written to: /home/noon/dev/ext/tensorflow-haskell/.stack-work/logs/tensorflow-core-ops-0.1.0.0.log

    Linking /home/noon/dev/ext/tensorflow-haskell/tensorflow-core-ops/.stack-work/dist/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/Cabal-1.22.5.0/setup/setup ...
    /home/noon/dev/ext/tensorflow-haskell/.stack-work/install/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/lts-6.2/7.10.3/lib/x86_64-linux-ghc-7.10.3/tensorflow-0.1.0.0-GSrYl0vSuJ3KqlFS0XZj4L/libHStensorflow-0.1.0.0-GSrYl0vSuJ3KqlFS0XZj4L.a(Raw.o):(.text+0x1d90): undefined reference to `TF_DeleteDeprecatedSession'
    /home/noon/dev/ext/tensorflow-haskell/.stack-work/install/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/lts-6.2/7.10.3/lib/x86_64-linux-ghc-7.10.3/tensorflow-0.1.0.0-GSrYl0vSuJ3KqlFS0XZj4L/libHStensorflow-0.1.0.0-GSrYl0vSuJ3KqlFS0XZj4L.a(Raw.o):(.text+0x1f28): undefined reference to `TF_CloseDeprecatedSession'
    /home/noon/dev/ext/tensorflow-haskell/.stack-work/install/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/lts-6.2/7.10.3/lib/x86_64-linux-ghc-7.10.3/tensorflow-0.1.0.0-GSrYl0vSuJ3KqlFS0XZj4L/libHStensorflow-0.1.0.0-GSrYl0vSuJ3KqlFS0XZj4L.a(Raw.o):(.text+0x20c0): undefined reference to `TF_NewDeprecatedSession'
```

i believe this isn't being caught by the CI because that uses a different docker base.  (it's not clear to me why that is, of course, and i don't think i can see those images on docker hub for whatever reason). that is, i successfully ran `./ci_build/outer_launch_tests.sh` on my machine.

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -1,6 +1,6 @@
 # Prepare the image with:
 #   docker build -t tensorflow/haskell:v0 docker
-FROM gcr.io/tensorflow/tensorflow:latest-devel
+FROM tensorflow/tensorflow:nightly-devel
 MAINTAINER TensorFlow authors <tensorflow-haskell@googlegroups.com>
 # Installs protoc and the libraries.
 RUN \

